### DISCUSSION TITLE
Charger occupancy

### DISCUSSION THREAD

**Discussion by Goule** [timdorr/tesla-api#736] [Q&A]
Hello,

Some services can get superchargers occupancy from the domain akamai-apigateway-charging-
ownership.tesla.com I'm not unable to determine the API path, is there someone who already did it ?

Thanks !

**Comment by superfloh247** [comment]
yes, the TeslaLogger for example, see https://github.com/bassmaster187/TeslaLogger/blob/master/TeslaLogger/NearbySuCService.cs
